src/cpuinfo_x86.c

```diff
@@ -70,16 +70,24 @@ uint32_t GetXCR0Eax(void) { return _xgetbv(0); }
 #error "Unsupported compiler, x86 cpuid requires either GCC, Clang or MSVC."
 #endif
 
+static Leaf CpuId(uint32_t leaf_id) {
```

Comment:
Is this used?

Reply:
Yes, but it can be replaced with CpuIdEx(leafid, 0). Then I can remove old CpuId function and rename CpuIdEx into CpuId.

Reply:
I wasn't sure what solution fits best here, so decided to make new function.

Reply:
Then I think it shouldn't be `static`

Reply:
But it used only in `cpuinfo_x86.c`.

Reply:
Ha my bad, sorry about the confusion.